timelock-api/src/main/conjure/timelock-management-api.yml

```diff
@@ -8,3 +8,8 @@ services:
       getNamespaces:
         http: POST /getNamespaces
         returns: set<string>
+
+      achieveConsensus:
+        http: POST /achieveConsensus
+        args:
+          namespaces: set<string>
```

Comment:
Might be useful to add Conjure docs here around how this operation handles leadership and/or failures.